providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdOptions.java

```diff
@@ -13,114 +13,147 @@
 import lombok.Builder;
 import lombok.Getter;
 
-/** FlagdOptions is a builder to build flagd provider options. */
+/**
+ * FlagdOptions is a builder to build flagd provider options.
+ */
 @Builder
 @Getter
 @SuppressWarnings("PMD.TooManyStaticImports")
 public class FlagdOptions {
 
-    /** flagd resolving type. */
+    /**
```

Comment:
hmm my spotless removed those extra lines, i need to check the configuration, i hope this is not a common theme

Reply:
nvm this was just an error on my side ;) sorry for all the  unnecessary changes, with making the javadoc multiline again